examples/custom-log-format/README.md

```diff
@@ -6,14 +6,16 @@ This example lets you set the log-format for NGINX using the configmap reosurce
 kind: ConfigMap
 apiVersion: v1
 metadata:
-  name: oss-nginx-ingress
+  name: nginx-config
+  namespace: nginx-ingress
 data:
```

Comment:
`;` at the end of the log format is new. But it is not needed. Could you remove it?